File: Arquetipo-Bancolombia-Heroe/domain/model/src/main/java/co/com/sofka/model/carta/Carta.java
package co.com.sofka.model.carta;
import lombok.*;

@Data
@Builder(toBuilder = true)
@NoArgsConstructor
@AllArgsConstructor
public class Carta {

    private String id;
    private String nombre;
    private Long valor;
    private String uri;
    private Boolean esOculta;

    @Override
    public String toString() {
        return "Carta{" +
                "id='" + id + '\'' +
                ", nombre='" + nombre + '\'' +
                ", valor=" + valor +
                ", uri='" + uri + '\'' +
                ", esOculta=" + esOculta +
                '}';
    }
}

File: frontend-hero/src/app/components/models/carta.ts

export class Carta {
    id?: string;
    nombre: string;
    valor: number;
    uri: string;
    esOculta: boolean
    constructor(id: string, nombre: string, valor: number, uri: string, esOculta:boolean){
        this.id = id;
        this.nombre = nombre;
        this.valor = valor;
        this.uri = uri;
        this.esOculta = esOculta;

    }
}